scripts/createDB.ts

```diff
@@ -3,50 +3,113 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-/**
- * NOTE: This script is only to be used by senior candidates
- */
-
 import Database from 'better-sqlite3'
 import { executeQuery } from '@datocms/cda-client'
 import 'dotenv/config'
+import { ApiResponse } from 'types'
 
 const query = `query {
-	allDepartments(first: 100) {
-		name
-		id
-		parent {
-			name
-			id
-		}
-	}
-
-	allPeople(first: 100) {
-		id
-		name
-		title
-		avatar {
-			url
-		}
-		department {
-			name
-		}
-	}
+ allDepartments(first: 100) {
+   name
+   id
+   parent {
+     id
+   }
+ }
+ allPeople(first: 100) {
+   id
+   name
+   avatar {
+     url
+   }
+   department {
+     name
+     id
+   }
+	title
+ }
 }`
 
-const DATO_API_TOKEN = process.env.DATO_API_TOKEN // the Dato API token is provided in your Discord channel
+const DATO_API_TOKEN = process.env.DATO_API_TOKEN
 
 async function main() {
-	// API Docs: https://github.com/WiseLibs/better-sqlite3/blob/master/docs/api.md
```

Comment:
I spent a bit of time with the docs but did consult AI for help on `db` commands to add to sqlite db